py3-ruamel-yaml-clib.yaml

```diff
@@ -28,13 +28,9 @@ pipeline:
       uri: https://files.pythonhosted.org/packages/source/r/ruamel.yaml.clib/ruamel.yaml.clib-${{package.version}}.tar.gz
       expected-sha256: beb2e0404003de9a4cab9753a8805a8fe9320ee6673136ed7f04255fe60bb512
 
+  - uses: py/pip-build-install
+
   - runs: |
-      python3 -m gpep517 build-wheel \
-        --wheel-dir dist \
-        --output-fd 3 3>&1 >&2
-      python3 -m installer \
-        -d "${{targets.destdir}}" \
-        dist/ruamel.yaml.clib-${{package.version}}-*.whl
       install -Dm644 LICENSE \
```

Comment:
Does `py/pip-build-install` not also copy the license?

Reply:
it does not copy it.